sdk/nodejs/connect.ts

```diff
@@ -7,32 +7,7 @@ import * as utilities from "./utilities";
 import {Facility} from "./index";
 
 /**
- * Provides a resource for [Packet Connect](https://www.packet.com/cloud/all-features/packet-connect/), a link between Packet VLANs and VLANs in other cloud providers.
- * 
- * ## Example Usage
- * 
- * ```typescript
- * import * as pulumi from "@pulumi/pulumi";
- * import * as packet from "@pulumi/packet";
- * 
- * const vlan1 = new packet.Vlan("vlan1", {
- *     description: "VLAN in New Jersey",
- *     facility: "ewr1",
- *     projectId: local_project_id,
- * });
- * const myExpressroute = new packet.Connect("myExpressroute", {
- *     facility: "ewr1",
- *     name: "test",
- *     portSpeed: 100,
- *     projectId: local_project_id,
- *     // provider ID for Azure ExpressRoute is ed5de8e0-77a9-4d3b-9de0-65281d3aa831
- *     providerId: "ed5de8e0-77a9-4d3b-9de0-65281d3aa831",
- *     // providerPayload for Azure ExpressRoute provider is your ExpressRoute
- *     // authorization key (in UUID format)
- *     providerPayload: "58b4ec12-af34-4435-5435-db3bde4a4b3a",
- *     vxlan: vlan1.vxlan,
- * });
- * ```
+ * Packet Connect was removed in release 2.7.0.
```

Comment:
Don't we have a mechanism to explicitly mark resources as deprecated?

Reply:
Not at this time... this is actually code that was removed in the upstream provider :/

Reply:
FYI ;) https://github.com/pulumi/pulumi-terraform-bridge/issues/86 I will be addressing this during 0.30